README.md

```diff
@@ -63,6 +63,38 @@ public class LogzioLogbackExample {
 }
 ```
 
+### MDC
+Each key value you will add to MDC will be added to each log line as long as the thread alive. No further configuration needed.
+```java
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+public class LogzioLogbackExample {
+
+    public static void main(String[] args) {
+        Logger logger = LoggerFactory.getLogger(LogzioLogbackExample.class);
+
+        MDC.put("Key", "Value");
+        logger.info("This log will hold the MDC data as well");
+    }
+}
+```
```

Comment:
I would put an example here of the JSON sent to emphasize it's added to the outgoing JSON.